examples/basic/doczrc.js

```diff
@@ -2,6 +2,8 @@ const { storybook } = require('docz-plugin-storybook')
 
 module.exports = {
   plugins: [
-    storybook()
+    storybook({
+      wrapper: require.resolve('./story-wrapper')
```

Comment:
let's call this `storyWrapper` since we could potentially have multiple wrappers and when in doubt, it's always better to have verbose, descriptive variable names.

my general rule of thumb is that if i'm reading this variable name and i would need a comment to figure out what it's supposed to do, then the name isn't precise enough.

Reply:
✅